modules/getting-started/con-audit-log-file-rotation-overview.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assembly:
+// assembly-audit-log.adoc
+
+[id="con-audit-log-file-rotation-overview_{context}"]
+= About audit log file rotation
+
+Logging to a rotating file in {product} is helpful for persistent storage for persistent audit logs.
+
+{product-short} supports audit logging for both the catalog and scaffolder. Audit logs can be identified by the `isAuditLog: true` field.
```

Comment:
There are more plugins that are supported besides catalog and scaffolder.

Reply:
OK. 

https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/audit-log.md#audit-logging-for-backstage-showcase says: 'Backstage Showcase supports audit logging for both the catalog and scaffolder. Audit logs can be identified by the isAuditLog: true field." 

Currently, https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/getting_started_with_red_hat_developer_hub/assembly-audit-log walks users through audit logs for OpenShift Container Platform, and only has catalog events https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/getting_started_with_red_hat_developer_hub/assembly-audit-log#ref-audit-log-catalog-events.adoc_assembly-audit-log, and scaffolder events https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/getting_started_with_red_hat_developer_hub/assembly-audit-log#ref-audit-log-scaffolder-events.adoc_assembly-audit-log

What else is supported, and where in Red Hat product docs are those plugins documented? I can update this accordingly with that information.

Reply:
Per Slack: update to say: {product-short} supports audit logging.